analyzers/src/SonarAnalyzer.CSharp/Rules/GenericReadonlyFieldPropertyAssignment.cs

```diff
@@ -62,45 +61,40 @@ protected override void Initialize(SonarAnalysisContext context)
                 SyntaxKind.RightShiftAssignmentExpression,
                 SyntaxKindEx.CoalesceAssignmentExpression);
 
-            context.RegisterSyntaxNodeActionInNonGenerated(
-                    c =>
```

Comment:
According to the coding style:
`Method invocation arguments should be placed on the same line only when they are few and simple. Otherwise, they should be placed on separate lines. The first argument should be on a separate line, aligned with a left-most single indendation.`
So the previous version where `c=>` was in a separate line was correct.

Reply:
One of the first things I learned when I was working with @pavel-mikula-sonarsource on the record structs, was to place the `c =>` on the previous line (see e.g. [my very first PR](https://github.com/SonarSource/sonar-dotnet/pull/5561/files#diff-3c288213df4c8bfb602b26670b405a1a9d7110ffdee3ec7d2549488b062dc514R61-R62)). I have no strong opinion on either style and both are used throughout the code base.
I would leave it as is for the moment and maybe discuss it in "how we work" or in slack.

Reply:
During the discussion about this topic, we said that having `c =>` on the previous line is fine. It doesn't improve anything, having it on the next line.

Reply:
I have put a bullet point on the agenda for the how we work discussion.